### ISSUE TITLE
PhpdocScalarFixer.php: Notice: Undefined offset: 1

### ISSUE THREAD

**Issue by acrozes** [FriendsOfPHP/PHP-CS-Fixer#1147]
Using last PHP-CS-Fixer on a symfony project with php 5.6: 

Notice: Undefined offset: 1 in phar://C:/[...]/php-cs-fixer.phar/Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php on line 92
PHP Notice:  Undefined offset: 1 in phar://C:/[...]/php-cs-fixer.phar/Symfony/CS/Fixer/Symfony/PhpdocScalarFixer.php on line 92

**Comment by GrahamCampbell** [comment]
We need to see the actual code you are trying to fix please.

**Comment by acrozes** [comment]
An example: 

```
namespace AC\DemoBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use JMS\Serializer\Annotation\Type as Type;

/**
 * Inscription.
 *
 * @ORM\Table(name="inscription",
 *            indexes={@ORM\Index(name="inscription_ref_idx",        columns={"ref"}),
 *                     @ORM\Index(name="inscription_ref_event_idx",  columns={"ref_event"}),
 *                     @ORM\Index(name="inscription_mail_idx",       columns={"mail"}),
 *                     @ORM\Index(name="inscription_id_univers_idx", columns={"id_univers"})
 *                    }  )
 * @ORM\Entity(repositoryClass="AC\DemoBundle\Entity\InscriptionRepository")
 */
class Inscription
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @Type("integer")
     */
    private $id;

    /**
     * @var int
     *
     * @ORM\Column(name="id_univers", type="integer")
     * @Type("integer")
     */
    private $id_univers;

    /**
     * @var int
     *
     * @ORM\Column(name="id_session", type="integer")
     * @Type("integer")
     */
    private $id_session;

    /**
     * @var string
     *
     * @ORM\Column(name="civilite", type="string", length=255)
     * @Type("string")
     */
    private $civilite;

[...]

**Comment by keradus** [comment]
`Tag` class do the lowercase on tag type. That is why it does not distinguish between `@type` and `@Type`.
@GrahamCampbell , why is that?

On the other hand we can different between simple tag (`@type`) and complex one (`@Type(foo)`)

**Comment by GrahamCampbell** [comment]
Hmmm, it's not that we don't distinguish between them, I don't think. I'll have a look at this in detail when I have some free time.

**Comment by keradus** [comment]
`$docBlock->getAnnotationsOfType('type')` returns both `@type` and `@Type(foo)`.

**Comment by GrahamCampbell** [comment]
Looks like a bug then, possibly due to the brackets? I can't remember exactly how I implemented this. As I said, I'll be able to look into this later. :)

**Comment by keradus** [comment]
As I said it is because the `Tag` class does the lowercase on tag type:
https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.7/Symfony/CS/DocBlock/Tag.php#L51
That how it is implemented ;)

**Comment by keradus** [comment]
ping @GrahamCampbell

**Comment by keradus** [comment]
ping @GrahamCampbell

**Comment by GrahamCampbell** [comment]
Sorry, I can't look into this right now. I will do when ever I get a chance.

**Comment by keradus** [comment]
ping @GrahamCampbell ;)

**Comment by keradus** [comment]
@acrozes Please see #1204 ;)

**Comment by keradus** [comment]
Closing due to merge of #1204

**Comment by acrozes** [comment]
I confirm it is fixed in v1.8.1
